Visual Studio/Realm.vcxproj

```diff
@@ -363,7 +363,8 @@
       <PrecompiledHeader>
       </PrecompiledHeader>
       <DisableSpecificWarnings>4355;4996</DisableSpecificWarnings>

```

Comment:
I think it was in the 3'rd party getopt.h file. I suggest not to alter 3'rd party files in any ways if not absolutely needed...